docs/developers_guide/landice/framework.rst

```diff
@@ -10,3 +10,96 @@ extrapolate
 The landice framework module ``compass/landice/extrapolate.py`` provides a
 function for extrapolating variables into undefined regions.  It is copied
 from a similar script in MPAS-Tools.
+
+mesh
+~~~~
+
+The landice framework module :py:mod:`compass.landice.mesh` provides functions
+used by all ``mesh_gen`` tests cases, which currently exist within the
+``antarctica``, ``greenland``, ``humboldt``, ``kangerlussuaq``, ``koge_bugt_s``,
+and ``thwaites`` test groups. These functions include:
+
+:py:func:`compass.landice.mesh.gridded_flood_fill()` applies a flood-fill algorithm
+to the gridded dataset in order to separate the ice sheet from peripheral ice.
+
+:py:func:`compass.landice.mesh.set_rectangular_geom_points_and_edges()` sets node
+and edge coordinates to pass to py:func:`mpas_tools.mesh.creation.build_mesh.build_planar_mesh()`.
+
+:py:func:`compass.landice.mesh.set_cell_width()` sets cell widths based on settings
+in config file to pass to :py:func:`mpas_tools.mesh.creation.build_mesh.build_planar_mesh()`.
+Requires the following options to be set in the given config section: ``min_spac``,
+``max_spac``, ``high_log_speed``, ``low_log_speed``, ``high_dist``, ``low_dist``,
+``high_dist_bed``, ``low_dist_bed``, ``high_bed``, ``low_bed``, ``cull_distance``,
+``use_speed``, ``use_dist_to_edge``, ``use_dist_to_grounding_line``, and ``use_bed``.
+
+:py:func:`compass.landice.mesh.get_dist_to_edge_and_GL()` calculates distance from
```

Comment:
```suggestion
:py:func:`compass.landice.mesh.get_dist_to_edge_and_gl()` calculates distance from
```

Reply:
resolved in [5543fa9](https://github.com/MPAS-Dev/compass/pull/590/commits/5543fa9aa0f2a1aef4d68f066d2683a9aba6b6a7)